app/controllers/self_service/abstract_firms_controller.rb

```diff
@@ -47,5 +51,12 @@ def principal
     def firm_params
       params.require(:firm).permit(*FIRM_PARAMS)
     end
+
+    def redirect_to_edit(firm_id: params[:firm_id])
+      redirect_to url_for(
```

Comment:
I'm wondering why this is ```url_for``` rather than ```path_for```. Is there not a url helper for this route?